### DISCUSSION TITLE
Details about used Fresnel Equations in rough conductor BRDF

### DISCUSSION THREAD

**Discussion by kobabulbul** [mitsuba-renderer/mitsuba3#799] [Q&A] [answered]
I want to know the used Fresnel Equations and the associated default parameters for:

bsdf = mi.load_dict({
'type': 'roughconductor',
'alpha': 0.2,
'distribution': 'ggx'
})

The documentation says it follows the routine of Microfacet Models for Refraction through Rough Surfaces by Walter et al. The presented Fresnel Equations there is for unpolarized dielectrics. Is it used in the model? 

How can I change the parameters?

I hope it's okay I pose this question here.

Best regards and thanks in advance.

**Comment by njroussel** [comment, marked as answer]
Hi @koba341 

Have you taken a look at [this page](https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_bsdfs.html#rough-conductor-material-roughconductor) yet? By default the material is set to `none` which means a relative IOR (`eta`) of 0 (and imaginary part `k` is 1). I think it's pretty easy to follow this logic in the [source code](https://github.com/mitsuba-renderer/mitsuba3/blob/master/src/bsdfs/roughconductor.cpp) too if that helps.

All of these parameters can be changed when instantiating the plugin with `load_dict` or when updating them with `mi.traverse()`. Refer to the tutorials/documentation for examples of that.

**Reply by kobabulbul** [reply]
Thank you very much, the source code was very straight forward and helpful. Indeed, the Fresnel Equations for Rough Conductors differ from the one in Walter et. al. 

Thank you also btw for making this renderer. I'm doing optical metrology in the thermal infrared and I think your python package can be very helpful. I try to use it for simple simulation purposes. I have the general problem that I don't exactly know what the commands do, but with the source code at hand it shouldn't be a big problem anymore.

Best regards,
Hoschang Noori
